.clang-tidy

```diff
@@ -16,6 +18,10 @@ Checks: '
     -modernize-avoid-c-arrays,
     -modernize-use-nodiscard,
     -modernize-use-trailing-return-type,
-  performance-*
+  performance-*,
+    -performance-avoid-endl
```

Comment:
Another report I find worthwhile disabling is the `misc-use-anonymous-namespace`. It suggests to convert

```c++
static void foo() {}
```

to

```c++
namespace {
void foo() {} 
}
```

Which I think it is a counterproductive suggestion, bad for readability reasons. If I see `foo()` and want to verify that it is a local symbol I first would have to go through half the file on average to find an opening anonymous namespace. `static` makes that immeidately clear.

I think c++ briefly deprecated the use of `static` for this use-case in c++11 (which is why this report exist), but it was un-deprecated later.

What _is_ much more useful is `misc-use-internal-linkage`, which appeared in clang-tidy 19 or 20. It finds places that  should be static/in anonymous namespace but isn't.

Reply:
I believe the case against `static` is that someone can `extern` a function which was intended to be local to the compilation unit. We somehow do that in a few places in OpenSCAD for unknown reasons.

Reply:
if it is `static`, it is equivalent to being in an anonymous namespace, i.e. it can not be referred to by `extern` somewhere else.

Reply:
Right, I managed to confuse myself again :) I'll do another global run and look at the `misc-use-anonymous-namespace` check results.